pkgs/openclarity/vmclarity/registry.yaml

```diff
@@ -0,0 +1,17 @@
+packages:
+  - type: github_release
+    repo_owner: openclarity
+    repo_name: vmclarity
+    description: VMClarity is an open source tool for agentless detection and management of Virtual Machine Software Bill Of Materials (SBOM) and security threats such as vulnerabilities, exploits, malware, rootkits, misconfigurations and leaked secrets
+    asset: vmclarity-cli-{{trimV .Version}}-{{.OS}}-{{.Arch}}.{{.Format}}
+    format: tar.gz
+    files:
+      - name: vmclarity
+        src: vmclarity-cli
```

Comment:
I guess the command name should be `vmclarity-cli` same as https://github.com/aquaproj/aqua-registry/pull/16386#discussion_r1363892432 .

Reply:
- https://github.com/openclarity/vmclarity/pull/829

Reply:
done

Reply:
Thank you, but could you stop the force push next time?
https://github.com/suzuki-shunsuke/oss-contribution-guide#dont-do-force-pushes-after-opening-pull-requests